### ISSUE TITLE
use npm for plotly

### ISSUE THREAD

**Issue by chantisnake** [WheatonCS/Lexos#697]
There are couple tools that uses plotly:
- current dendrogram.
- future RWA: https://github.com/WheatonCS/Lexos/tree/feature/rwa_model
- future stats: https://github.com/WheatonCS/Lexos/tree/feature/stats_model

However, currently we directly load plotly from CDN:
- dendrogram:
https://github.com/WheatonCS/Lexos/blob/c6900093bff7d7855a0cd704421378a6c5767459/lexos/templates/dendrogram.html#L7
- RWA:
https://github.com/WheatonCS/Lexos/blob/a5ebcf4897f7633f34a67359118aedd766257a99/lexos/templates/rwanalysis.html#L8
- stats:
https://github.com/WheatonCS/Lexos/blob/05aeca570c6257e77eee64919d7c0461365a2370/lexos/templates/statistics.html#L13

These are not consistent with our frontend library management. Therefore we need to change it to use npm to manage package.

**Comment by scottkleinman** [comment]
Yes, this is a leftover from our experiments with Plotly last year. We definitely need to have a local copy.

**Comment by scottkleinman** [comment]
Just in case people are not aware, there is a [Library Maintenance Guide](https://github.com/WheatonCS/Lexos/blob/master/lexos/static/library_maintenance_guide.md) for Javascript. It may be helpful.
